ejp_xml_pipeline/transform_zip_xml/ejp_zip.py

```diff
@@ -49,12 +50,15 @@ def join_zip_and_xml_filename(zip_filename, xml_filename):
 
 
 def iter_parse_xml_in_zip(
-        zip_file: ZipFile, zip_filename: str,
+        zip_file: ZipFile, zip_filename: str, xml_filename_exclusion_regex_pattern : str
 ) -> Iterable[ParsedDocument]:
     imported_timestamp_str = format_to_iso_timestamp(datetime.now())
     zip_manifest = parse_go_xml(parse_zip_xml_root(zip_file, 'go.xml'))
     filenames = zip_manifest.filenames
     for filename in filenames:
+        if re.match(xml_filename_exclusion_regex_pattern, filename):
```

Comment:
might be good to be able to disable this using an empty or absent regular expression.